src/backend/tests/Fusion.Resources.Api.Tests/IntegrationTests/InternalRequests/DirectRequestTests.cs

```diff
@@ -223,6 +267,27 @@ public async Task DirectRequest_Create_ShouldBeAbleToProposePersonDirectly()
             response.Value.ProposedPerson?.Person.Mail.Should().Be(proposedPerson.Mail);
         }
 
+        [Fact]
+        public async Task DirectRequest_Create_ShouldBeBadRequest_WhenDirectRequestRequired()
+        {
+            using var adminScope = fixture.AdminScope();
+            var directRequestBasePosition = testProject.AddBasePosition("Direct request test", s =>
+            {
+                s.Settings["requireDirectRequest"] = true;
+            });
+
+            var position = testProject.AddPosition(p => p.BasePosition = directRequestBasePosition);
+            var proposedPerson = fixture.AddProfile(FusionAccountType.Employee);
+
+            var response = await Client.TestClientPostAsync<TestApiInternalRequestModel>($"/projects/{projectId}/requests", new
+            {
+                type = "normal",
+                orgPositionId = position.Id,
```

Comment:
When you don't provide a subType, the proper subtype will be resolved(in ResolveSubType).
To make this test green, you have to provide an invalid subType